.github/workflows/upgrade_test.yml

```diff
@@ -0,0 +1,58 @@
+name: Run upgrade test
+on:
+    workflow_dispatch:
+jobs:
+  test:
+    name: Run upgrade test
+    runs-on: ubuntu-latest
+    env:
+      AZURE_TENANT_ID: ${{ secrets.TenantId }}
+      AZURE_SUBSCRIPTION_ID: ${{ secrets.SubscriptionId }}
+      AZURE_CLIENT_ID: ${{ secrets.ClientId }}
+      AZURE_CLIENT_SECRET: ${{ secrets.ClientSecret }}
+      AZURE_LOCATION: westus2
```

Comment:
westus2 is pretty heavily used and we can't easily get more quota. Can you use another region instead (e.g. westus, westus3, eastus, westeurope)?